webapp/src/components/common/GroupedDataTable/index.tsx

```diff
@@ -44,6 +43,10 @@ function GroupedDataTable<TData extends TRow>({
   const [tableData, setTableData] = useState(data);
   const [rowSelection, setRowSelection] = useState<MRT_RowSelectionState>({});
 
+  useEffect(() => {
```

Comment:
Remove this useEffect, the table must have the same behavior as a form, the value is set only at first rendering and is persisted